### PR TITLE
Update and clean up The Team listing

### PR DESCRIPTION
**Pull request by pishifat**

### CODE REVIEW

**Comment by cl8n** [comment]
while ur at it could u clean up the listing format for the rest of these (some use "and", some don't, and one of them has "&")

pick one they're all fine

**Comment by pishifat** [comment]
yes sir

**Comment by peppy** [comment]
`, and` :(

File: wiki/People/The_Team/en.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ The people listed below are the core of the osu!team, and are chiefly responsibl
 | ![][flag_GB] [-Mo-](https://osu.ppy.sh/users/2202163) | Modding and mapping scene advisor |
 | ![][flag_JP] [nanaya](https://osu.ppy.sh/users/2387883) | osu!web developer |
 | ![][flag_PH] [Nathanael](https://osu.ppy.sh/users/2295078) | General housekeeping and moderation advisor |
-| ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) | osu!web developer & [official osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
+| ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) | osu!web developer and [official osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
 | ![][flag_JP] [notbakaneko](https://osu.ppy.sh/users/10751776) | osu!web developer |
-| ![][flag_US] [Nyquill](https://osu.ppy.sh/users/682935) | osu! development, Mappers' Guild organisation |
+| ![][flag_US] [Nyquill](https://osu.ppy.sh/users/682935) | osu! development |
```
Comment (peppy) [inline wiki/People/The_Team/en.md]: inactive

**Comment by cl8n** [comment]
descriptions were inconsistent (between things like "maintenance" and "maintainer") so I wrote them all as roles instead

**Comment by cl8n** [comment]
repflez hasn't been doing "wiki maintainer" work for a while but I'll keep for now since idk if he has plans to come back to it

**Pull request merged**